reference/modules/ROOT/pages/typeql/patterns/index.adoc

```diff
@@ -3,7 +3,7 @@
 This reference covers the usage of patterns in TypeQL.
 
 == Scopes and operations
-
+// TODO: Must update.
```

Comment:
TODO: This is actually the ideal state but I'm not sure we'll ever implement it.